content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -216,14 +216,14 @@ you can create it by running `sudo mkdir -m 755 /etc/apt/keyrings`
    sudo sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
    ```
 
-   {{< caution >}}
-   - Setting SELinux in permissive mode by running `setenforce 0` and `sed ...`
-     effectively disables it. This is required to allow containers to access the host
-     filesystem; for example, some cluster network plugins require that. You have to
-     do this until SELinux support is improved in the kubelet.
-   - You can leave SELinux enabled if you know how to configure it but it may require
-     settings that are not supported by kubeadm.
-   {{< /caution >}}
+{{< caution >}}
+- Setting SELinux in permissive mode by running `setenforce 0` and `sed ...`
+  effectively disables it. This is required to allow containers to access the host
+  filesystem; for example, some cluster network plugins require that. You have to
+  do this until SELinux support is improved in the kubelet.
+- You can leave SELinux enabled if you know how to configure it but it may require
+  settings that are not supported by kubeadm.
+{{< /caution >}}
```

Comment:
**untested**
Try changing the tab shortcode that contains this from `{{% tab name="Red Hat-based distributions" %}}` to `{{< tab name="Red Hat-based distributions" >}}`

Reply:
The [preview](https://deploy-preview-43432--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl) looks fine now.

If my understanding is correct, I'm afraid that simply picking and changing `{{% tab name="Red Hat-based distributions" %}}` in the following context may cause some errors:

```
{{< tabs name="k8s_install" >}}

{{% tab name="Debian-based distributions" %}}
{{% /tab %}}

{{% tab name="Red Hat-based distributions" %}}
{{% /tab %}}

{{% tab name="Without a package manager" %}}
{{% /tab %}}

{{< /tabs >}}
```

Let me have a try. If it doesn't work, revert it.

Reply:
Oops, only changing `{{< tab name="Red Hat-based distributions" >}}` doesn't work. Will change it back.